api/v1alpha1/clusterinstance_types.go

```diff
@@ -366,15 +373,15 @@ type ClusterInstanceStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Conditions",xDescriptors={"urn:alm:descriptor:io.kubernetes.conditions"}
```

Comment:
Was this important at all?

Reply:
Nope, got it from somewhere else. Cleaned it up now!